src/regressor.jl

```diff
@@ -137,6 +137,9 @@ function MLJBase.update(model::Union{NeuralNetworkRegressor, MultivariateNeuralN
         chain = old_chain
         epochs = model.n - old_model.n
     else
+        n = MLJBase.schema(X).names |> length
+        m = length(y[1])
```

Comment:
for consistency maybe use the same code as for the classifier. Btw you can also pass `y[1]` instead of `levels(y)` in the return of the `fit`

Reply:
The levels can then be obtained with `classes(y[1])`.